src/components/AnchorForCommentsOnly/anchorContextMenuOptions.js

```diff
@@ -0,0 +1,13 @@
+import Clipboard from '../../libs/Clipboard';
+import {Clipboard as ClipboardIcon, Checkmark} from '../Icon/Expensicons';
+
+
+export default function anchorContextMenuOptions(href, translate) {
```

Comment:
I think that the list of context actions will need to be dynamic depending on the context. So by default it will be the existing menu, but if we longPress on a link, we'll show "Copy URL to Clipboard".

Reply:
So yeah I think we're on the same page here 🙃

_side note:_ `ContextMenuActions.js` should be renamed `contextMenuActions.js` because it doesn't export a React component

Reply:
yeah. My idea was to use the `shouldShow` flag on actions and manage the actions on the context menu. So we just need to pass the keys to actions. Something like that.